internal/pkg/logging/workload.go

```diff
@@ -173,10 +177,11 @@ func (s *WorkloadClient) WriteLogEvents(opts WriteLogEventsOpts) error {
 	if opts.IncludeStateMachineLogs {
 		logStreamLimit *= 2
 	}
-	// TODO: there should be a separate logging client for ECS services
+	// TODO(lou1415926): there should be a separate logging client for ECS services
 	// and App Runner services. This `if` check is only ever true for ECS services.
+	// Refactor so that there are separate client for rdws, job and other services.
 	if s.logStreamNamePrefix != "" {
```

Comment:
Isn't `s.logStreamNamePrefix` only used as a marker value now after this PR? I think it could probably be replaced by some sort of boolean since its content (other than the fact of it being non-empty) is ignored now.

Reply:
+1 maybe will be included in this pending refactoring? https://github.com/aws/copilot-cli/pull/3895/files#diff-5b02f52c7202c54f55f56d2003862894a82400445cfa061e6069ddfeb0c2f8a0R180

Reply:
Yes I expect that variable to be refactored out as part of the `TODO` refactor. But I changed it to bool and renamed it anyway for readability before I finish the refactor/